src/m_misc.c

```diff
@@ -3222,25 +3222,36 @@ void M_LoadDefaults (void)
   }
 
   NormalizeSlashes(defaultfile);
-  I_Printf(VB_INFO, " default file: %s", defaultfile);
 
   // read the file in, overriding any set defaults
   //
   // killough 9/21/98: Print warning if file missing, and use fgets for reading
 
-  if (!(f = M_fopen(defaultfile, "r")))
-    I_Printf(VB_WARNING, "Warning: Cannot read %s -- using built-in defaults",defaultfile);
-  else
-    {
-      char s[256];
+  if ((f = M_fopen(defaultfile, "r")))
+  {
+    char s[256];
 
-      while (fgets(s, sizeof s, f))
-        M_ParseOption(s, false);
-      fclose (f);
-    }
+    while (fgets(s, sizeof s, f))
+      M_ParseOption(s, false);
+  }
+
+  I_InitPrintf();
```

Comment:
Sure, I like moving `I_InitPrintf();` to `M_LoadDefaults()`. I didn't do that because we have everything initialized in `D_Main()`. It might be worth adding a comment about this.